build/get-dotnet.sh

```diff
@@ -18,7 +18,7 @@ ensure_dir() {
 # main
 
 # resolve SDK version
-sdk_version="8.0.100-rc.1.23407.2"
```

Comment:
I don't know what this does.

Reply:
It reads the SDK version from global.json